docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md

```diff
@@ -63,17 +63,16 @@ If the universe has [encryption at rest enabled](../../../security/enable-encryp
 
 For information on how to schedule backups for a later time or as a recurring task, see [Schedule universe YCQL data backups](../../schedule-data-backups/ycql/).
 
-To view detailed information about an existing backup, click on it to open **Backup Details**. In addition to actions such as deleting and restoring the backup, as well as restoring tables and copying their locations, you can use **Backup Details** to add an incremental backup for universes that had the YB Controller automatically installed during their creation. To do so, click **Add Incremental Backup** to open the dialog shown in the following illustration:
+To view detailed information about an existing backup, click on it to open **Backup Details** shown in the following illustration:<br>
 
-![Incremental Backup](/images/yp/backup-increment-ycql.png)
-
-In addition, YB Controller-enabled universes can have their throttle parameters configured for more efficient backups and restore. For more information, see [Back up data universe YSQL data](../../back-up-universe-data/ysql/).
+![Backup details](/images/yp/backup-details-ycql.png)
 
-<br><br>To access a list of all backups from all universes, including the deleted universes, navigate to **Backups** on the YugabyteDB Anywhere left-side menu, as per the following illustration:
+In addition to actions such as deleting and restoring the backup, as well as restoring tables and copying their locations, you can use **Backup Details** to add an incremental backup for universes that had the YB Controller automatically installed during their creation. To do so, click **Add Incremental Backup** and complete the dialog shown in the following illustration:
```

Comment:
Same as my other comment, no longer this is popped up to be modified.

Reply:
<img width="1792" alt="image (2)" src="https://user-images.githubusercontent.com/42805130/209162037-3a1e180b-2d5e-42f0-8cb5-64bfd5b77600.png">

Reply:
The above image would be apt

Reply:
Changed the last sentence to:

To do so, click **Add Incremental Backup**, and then click **Confirm** on the **Add Incremental Backup** dialog.

I'm not including the dialog image because it doesn't bring any additional value to the reader.